src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -176,42 +176,13 @@ inline int64_t file_size(const char* path) {
  * @param[in]  path  The file name
  * @return     file size
  */
-inline bool file_exists(const char* path) {
```

Comment:
I will recommend to refactor this util function first and test it (as separate PR).
There is specific handing for android paths, I'm not sure if std implementation will work the same.